creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -160,7 +160,8 @@ const handleSyncFromPrimary = async ({
       selfEndpoint: thisContentNodeEndpoint,
       logger
     })
-    if (!_.isEmpty(error)) {
+
+    if (error) {
```

Comment:
why this change? i think [lodash method is safer](https://lodash.com/docs/4.17.15#isEmpty)

Reply:
this is ok with typing